lib/ansible/modules/cloud/azure/azure_rm_securitygroup.py

```diff
@@ -87,11 +87,15 @@
                 default: "*"
             source_address_prefix:
                 description:
-                  - IP address or CIDR from which traffic originates.
+                  - The CIDR or source IP range. Asterix '*' can also be used to match all source IPs. Default tags such
```

Comment:
It is best to split each sentence up as a new list entry, e.g.

```
- The CIDR or source IP range.
- Asterix ...
- ...
```

When referring to values, enclosing them in `C()`, like `C(*)`, makes them highlighted in the actual docs so is a better choice than just quoting them. Not related to this but you can also use `I()` when referencing other options.